autotest/gcore/vsigs.py

```diff
@@ -983,18 +983,15 @@ def method(request):
     gdal.SetConfigOption('GOA2_AUTH_URL_TOKEN', None)
     gdal.Unlink('/vsimem/.boto')
 
+
 ###############################################################################
 # Read credentials from simulated GCE instance
-
-
+@pytest.mark.skipif(sys.platform not in ('linux', 'linux2', 'win32'), reason='Incorrect platform')
```

Comment:
`linuxN` was dropped from Py3.3+

```suggestion
@pytest.mark.skipif(sys.platform not in ('linux', 'win32'), reason='Incorrect platform')
```